src/test/java/org/folio/rest/api/ItemDamagedStatusAPITest.java

```diff
@@ -0,0 +1,247 @@
+      })
+      .thenCompose(response -> {
+        status.setName("second test item damaged status name");
+        return client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT);
+      })
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      })
+      .thenCompose(it -> client.get(itemDamagedStatusesUrl(EMPTY), TEST_TENANT))
+      .whenComplete((response, throwable) -> {
+        int size = response.getJson().getJsonArray("itemDamageStatuses").size();
+        assertThat(size, is(2));
+      });
+  }
+
+  @Test
+  public void shouldFindItemDamagedStatusesByName() {
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      })
+      .thenCompose(response -> {
+        status.setName("itemDamagedStatusName");
+        return client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT);
+      })
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      })
+      .thenCompose(it -> {
+        URL url = itemDamagedStatusesUrl("?query=name=itemDamagedStatusName");
+        return client.get(url, TEST_TENANT);
+      })
+      .whenComplete((response, throwable) -> {
+        JsonObject body = response.getJson();
+        int size = body.getJsonArray("itemDamageStatuses").size();
+        assertThat(size, is(1));
+        JsonObject statuses = body.getJsonArray("itemDamageStatuses").getJsonObject(0);
+        assertThat(statuses.getString("name"), is("itemDamagedStatusName"));
+      });
+  }
+
+  @Test
+  public void canDeleteItemDamagedStatus() {
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      })
+      .thenCompose(response -> {
+        String id = response.getJson().getString("id");
+        return client.delete(itemDamagedStatusesUrl("/" + id), TEST_TENANT);
+      })
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_NO_CONTENT.toInt()));
+      });
+  }
+
+  @Test
+  public void cannotDeleteItemDamagedStatusThatDoesNotExist() {
+
```

Comment:
I think this blank line can be removed, other new tests don't have a leading blank line

Reply:
Done